src/infra/alarm/template.yaml

```
@@ -0,0 +1,60 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+
+Description: >-
+  Account Interventions Service's Alarm stack for ID Reuse.
```

```suggestion
  Alarm template for the Account Interventions Service solution.
```

Keeps consistent with main template description